package/aws/certificateauthoritypermission/composition.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: apiextensions.crossplane.io/v1
+kind: Composition
+metadata:
+  name: xawscertificateauthoritypermissions.certs.crossplane.dfds.cloud
+  labels:
+    provider: aws
+spec:
+  compositeTypeRef:
+    apiVersion: certs.crossplane.dfds.cloud/v1alpha1
+    kind: XAWSCertificateAuthorityPermission
+
+  patchSets:
+  - name: configname
+    patches:
+    - fromFieldPath: spec.claimRef.namespace
+      toFieldPath: spec.providerConfigRef.name
+      transforms:
+      - type: string
+        string:
+          fmt: "%s-aws"
+      policy:
+        fromFieldPath: Required
+  resources:
+  - name: certificateauthoritypermission
+    base:
+      apiVersion: acmpca.aws.crossplane.io/v1beta1
+      kind: CertificateAuthorityPermission
+      spec:
+        forProvider:
+          region: eu-west-1
+    patches:
+    - type: PatchSet
+      patchSetName: configname
+    - fromFieldPath: spec.parameters
+      toFieldPath: spec.forProvider
+    - type: ToCompositeFieldPath
+      fromFieldPath: "metadata.name"
+      toFieldPath: "status.createdResources.certificateauthoritypermission"
+    - type: FromCompositeFieldPath
+      fromFieldPath: "metadata.annotations[crossplane.io/external-name]"
+    - fromFieldPath: spec.parameters.deletionPolicy
+      toFieldPath: spec.deletionPolicy    
+    - type: ToCompositeFieldPath
+      fromFieldPath: status.conditions
+      toFieldPath: status.instanceConditions
+      policy:
+        fromFieldPath: Optional
+
+  - name: rbac
+    base:
+      apiVersion: crossplane.dfds.cloud/v1alpha1
+      kind: XRBAC
+      spec:
+        resourceTypes:
+        - certificateauthoritypermissions
+        apiGroups:
+        - certificateauthoritypermissions.acmpca.aws.crossplane.io

```

Comment:
shouldn't be just acmpca.aws.crossplane.io ?

Reply:
yes, well spotted. amended